crypto/multikey/kind_test.go

```diff
@@ -117,7 +117,7 @@ func TestKeyKindFromSignature(t *testing.T) {
 			"success-secp256k1-charlotte-key",
 			args{
 				func() []byte {
-					k, _ := secp256k1.PrivateKeyFromHex("DF4BA9F6106AD2846472F759476535E55C5805D8337DF5A11C3B139F438B98B3")
+					k, _ := secp256k1.PrivateKeyFromBytes([]byte("DF4BA9F6106AD2846472F759476535E55C5805D8337DF5A11C3B139F438B98B3"))
```

Comment:
`[]byte(string)` will attempt a literal conversation not work as expected as the `[]byte` is encoded as a hex string. For testing you can use the `encodingtest.MustDecodeHex` e.g. replace `[]byte` with `encodingtest.MustDecodeHex`.